pubsub/bigquery-controller.yaml

```diff
@@ -1,38 +1,32 @@
-id: bigquery-controller
-apiVersion: v1beta1
+apiVersion: v1
 kind: ReplicationController
-desiredState:
-  replicas: 2
-  # replicaSelector identifies the set of Pods that this
-  # replicaController is responsible for managing
-  replicaSelector:
+metadata:
+  labels:
     name: bigquery-controller
-  # podTemplate defines the 'cookie cutter' used for creating
-  # new pods when necessary
-  podTemplate:
-    desiredState:
-      manifest:
-        version: v1beta1
-        id: bq-controller
-        containers:
-          - name: bigquery
-            # Change this to point to your docker image
-            image: gcr.io/your-project/your-image-tag
-            env:
-              - name: PROCESSINGSCRIPT
-                value: pubsub-to-bigquery
-              # Change this to your pubsub topic
-              - name: PUBSUB_TOPIC
-                value: projects/your-project/topics/your-topic
-              # Change this to your project ID.
-              - name: PROJECT_ID
-                value: xxxx
-              # Change the following two settings to your dataset and table.
-              - name: BQ_DATASET
-                value: xxxx
-              - name: BQ_TABLE
-                value: xxxx
-    # Important: these labels need to match the selector above
-    # The api server enforces this constraint.
-    labels:
-      name: bigquery-controller
+  name: bigquery-controller
+spec:
+  replicas: 1
+  selector:
+    name: bigquery-controller
```

Comment:
The selector is redundant now, and is picked up automatically from the template labels. Unless you wanted specific differences.